include/tsv/tsv_except.hpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2015 Adrien Devresse <adrien.devresse@epfl.ch>
```

Comment:
This is not accurate :)

Reply:
Should it be `2019 Blue Brain Project` ?

Reply:
Yes, and Adrien didn't write this file, did he?

Reply:
Shall I change this in every file?

Reply:
I think the new ones are enough (for me, at least)

Reply:
Okay!